docs/release-notes/flink-1.9.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Release Notes - Flink 1.9"
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+These release notes discuss important aspects, such as configuration, behavior,
+or dependencies, that changed between Flink 1.8 and Flink 1.9. It also provides an overview
+on known shortcoming or limitations with new experimental features introduced in 1.9.
+
+Please read these notes carefully if you are planning to upgrade your Flink version to 1.9.
+
+* This will be replaced by the TOC
+{:toc}
+
+## Known shortcomings or limitations for new features
+
+### New Table / SQL Blink runner
+
+TBD
```

Comment:
Flink 1.9.0 support multiple runners under Table API, named `old planner` and `blink planner`. The `old planner` keeps the same behavior with Flink 1.8, but `blink planner` used a quite different architecture. Hence `blink planner` in 1.9.0 is still considered to be an experimental feature and also have some limitations:
- Blink planner can not be used with `BatchTableEnvironment`, hence the table program ran with blink planner can not transformed to `DataSet`. This is by design and will also not support in the future. 
- If you want to run batch job with blink planner, please use `TableEnvironment` only. 
- If you want to run streaming job with blink planner, you can use `TableEnvironment` or `StreamTableEnvironment`, depending on if you want to convert between `DataStream`.

Other known issues:
 - [FLINK-13708] The `TableEnvironment` instanced should not be reused if you want to execute multiple SQL statements with blink planner.

Reply:
Here is the list of hive limitations:
Hive limitations:
- INSERT OVERWRITE is not supported.
- Inserting into partitioned tables is not supported.
- ACID tables are not supported.
- Bucketed tables are not supported.
- Views are not supported.
- Only a limited number of table storage formats are supported, namely text, SequenceFile, ORC, and Parquet.

Other known issues:
- [FLINK-13534: Unable to query Hive table with decimal column](https://issues.apache.org/jira/browse/FLINK-13534)

Reply:
DDL

In Flink 1.9.0, we also added a preview feature about SQL DDL, but only for batch style DDLs. So all the streaming related concepts are not supported yet, like watermark.

Related issues:
- [FLINK-13661: Add a stream specific  CREATE TABLE SQL DDL]
- [FLINK-13568: DDL create table doesn't allow STRING data type]

Reply:
Add one more issue for DDL

- [FLINK-13699](https://issues.apache.org/jira/browse/FLINK-13699): TIMESTAMP/DATE/TIME types in DDL are not correctly recognized by blink planner.

Reply:
Add one more limitation for blink planner

- The implementation of `StreamTableSink` should implement `consumeDataStream` method instead of `emitDataStream` if it is used in blink planner. However, both methods work in old planner. This is by design to make the returned `DataStreamSink` accessible for planner.

Reply:
Should we also add a sentence to describe not all the documentation of blink planner is available? 

- [FLINK-13271](https://issues.apache.org/jira/browse/FLINK-13271): we are still working to add documentation for blink planner.